lib/runner/extensions/event.command.js

```diff
@@ -516,6 +543,8 @@ module.exports = {
                     // now that this script is done executing, we trigger the event and move to the next script
                     this.triggers.script(err || null, scriptCursor, result, script, event, item);
 
+                    shouldSkipExecution = isExecutionSkipped(executionId);
```

Comment:
This can be set to true inside the skip event listener itself.

Reply:
Done